server/page/user_content.go

```diff
@@ -11,6 +11,22 @@ import (
 	"strings"
 )
 
+func MakeLogoutHandler(ses *session.Sessions, strg *storage.Storage) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		LogoutHandler(w, r, ses, strg)
+	}
+}
+
+func LogoutHandler(w http.ResponseWriter, r *http.Request, ses *session.Sessions, strg *storage.Storage) {
+	sessionCookie, err := r.Cookie(session.SessionCookie)
+	if err == nil {
+		ses.Logout(sessionCookie.Value)
+	}
+
+	http.Redirect(w, r, "/active", http.StatusSeeOther)
+	ActiveSection(w, r, ses, strg)
```

Comment:
A call to `ActiveSection` isn't needed here, the call to `http.Redirect` already covers us here.
Please also remove the dependency on `storage.Storage` in the handler.